tests/integration/test_github.py

```diff
@@ -194,6 +194,14 @@ def test_pr_info(self):
         assert pr_info.title.startswith("Add basic structure")
         assert pr_info.status == PRStatus.closed
 
+    def test_update_pr_info(self):
+        self.colin_project.update_pr_info(
+            pr_id=1, title="new", description="new description"
+        )
```

Comment:
It can be hard to regenerate the response files for this test.

(When you run it multiple times, the `title` and `description` is not changed.)

What about changing the title/description two times so we are sure, the change was done?